src/Rules/Properties/LazyAlwaysUsedClassConstantsExtensionProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Properties;
+
+use PHPStan\DependencyInjection\Container;
+
+class LazyAlwaysUsedClassConstantsExtensionProvider implements AlwaysUsedClassConstantsExtensionProvider
+{
+
+	private Container $container;
+
+	/** @var AlwaysUsedClassConstantsExtension[]|null */
+	private $extensions = null;
```

Comment:
Can have a native `?array` type (PHAR compiler removes it automatically for PHP 7.1 support)